docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+version: '2'
+services:
+  db:
+    image: mongo
+  web:
```

Comment:
you can (should) define an `environment` section under web, something like:
```
environment:
 - MONGO_URI=mongodb://db
```
so that anybody running the docker-compose version will automatically talk to the mongo instance that is brought up by docker-compose.